drivers/wifi/esp32/src/esp_wifi_drv.c

```diff
@@ -198,41 +420,31 @@ static int eth_esp32_dev_init(const struct device *dev)
 	wifi_init_config_t config = WIFI_INIT_CONFIG_DEFAULT();
 	esp_err_t ret = esp_wifi_init(&config);
 
-	ret |= esp_supplicant_init();
+	ret |= esp_wifi_set_mode(WIFI_MODE_STA);
 	ret |= esp_wifi_start();
 
-	if (IS_ENABLED(CONFIG_ESP32_WIFI_STA_AUTO)) {
```

Comment:
Kconfig option should be removed with the same commit. I guess that `CONFIG_ESP32_WIFI_SSID` should be removed as well, or is it still used somewhere?

Reply:
`CONFIG_ESP32_WIFI_SSID` removed.